util/os-runner/install-dmd.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+wget https://netcologne.dl.sourceforge.net/project/d-apt/files/d-apt.list -O /etc/apt/sources.list.d/d-apt.list
```

Comment:
tbh I think installing with `.deb` would be more simple.
@teodutu what do you think since you are the master of D?

Reply:
You phrased it like this intentionally.

Reply:
This variant dynamically installs the latest version of `dmd`. AFAIK the `.deb` way only installs a given version, right? If so, I'd rather keep this approach.